templates/redis.conf.epp

```diff
@@ -1,74 +1,74 @@
-  Optional[String[1]]                     $notify_keyspace_events,
-  Integer[0]                              $hash_max_ziplist_entries,
-  Integer[0]                              $hash_max_ziplist_value,
-  Integer[0]                              $list_max_ziplist_entries,
-  Integer[0]                              $list_max_ziplist_value,
-  Integer[0]                              $set_max_intset_entries,
-  Integer[0]                              $zset_max_ziplist_entries,
-  Integer[0]                              $zset_max_ziplist_value,
-  Integer[0]                              $hll_sparse_max_bytes,
-  Boolean                                 $activerehashing,
-  String[1]                               $output_buffer_limit_slave,
-  String[1]                               $output_buffer_limit_pubsub,
-  Integer[1, 500]                         $hz,
-  Boolean                                 $aof_rewrite_incremental_fsync,
-  Boolean                                 $cluster_enabled,
-  String[1]                               $cluster_config_file,
-  Integer[1]                              $cluster_node_timeout,
-  Integer[0]                              $cluster_slave_validity_factor,
-  Boolean                                 $cluster_require_full_coverage,
-  Integer[0]                              $cluster_migration_barrier,
-  Optional[String]                        $extra_config_file,
+  Boolean                                            $daemonize,
+  Stdlib::Absolutepath                               $pid_file,
+  Boolean                                            $protected_mode,
+  Stdlib::Port                                       $port,
+  Integer[0]                                         $tcp_backlog,
+  Array[Stdlib::IP::Address]                         $bind_arr,
+  Variant[Stdlib::Absolutepath, Enum['']]            $unixsocket,
+  Variant[Stdlib::Filemode, Enum['']]                $unixsocketperm,
+  Integer[0]                                         $timeout,
+  Integer[0]                                         $tcp_keepalive,
+  Redis::LogLevel                                    $log_level,
+  Stdlib::Absolutepath                               $log_file,
+  Boolean                                            $syslog_enabled,
+  Optional[String[1]]                                $syslog_facility,
+  Integer[1]                                         $databases,
+  Boolean                                            $save_db_to_disk,
+  Hash                                               $save_db_to_disk_interval,
+  Boolean                                            $stop_writes_on_bgsave_error,
+  Boolean                                            $rdbcompression,
+  Variant[String[1], Boolean]                        $dbfilename,
+  Stdlib::Absolutepath                               $workdir,
+  Optional[String[1]]                                $slaveof,
+  Optional[Variant[String[1], Sensitive[String[1]]]] $masterauth,
```

Comment:
I think you're missing the unwrap for this that got lose in a rebase.

Reply:
No, in EPP you don't need `unwrap`.  Thats the Point.